pkg/local_object_storage/engine/put.go

```diff
@@ -213,7 +211,7 @@ func (e *StorageEngine) putToShard(sh shardWrapper, ind int, pool util.WorkerPoo
 
 	<-exitCh
 
-	return putSuccess, alreadyExists, overloaded
+	return alreadyExists, overloaded, putError
```

Comment:
`putError == nil` on `overload`. Seems like some callers can go false-positive with this

Reply:
True. Nice catch. Frankly, we can rework this all into a single error returned, would be much easier this way.